services/github/src/api.js

```diff
@@ -449,6 +456,96 @@ builder.declare({
   });
 });
 
+builder.declare({
+  method: 'post',
+  route: '/builds/:owner/:repo/cancel',
+  name: 'cancelBuilds',
+  scopes: 'github:cancel-builds:<owner>:<repo>',
```

Comment:
in order to use parameters in `scopes`, we have to include parameters with the same name to the `route`

This is probably less ideal, as it provides now a different url signature (unlike `.builds()`) but also requires both owner and repo (unlike `builds()`)